server/controllers/board.js

```diff
@@ -0,0 +1,45 @@
+const User = require("../models/User");
+const Board = require("../models/Board");
+const Column = require("../models/Column");
+const Card = require("../models/Card");
+const asyncHandler = require("express-async-handler");
+const generateToken = require("../utils/generateToken");
+
+// Create new board
+exports.createBoard = asyncHandler(async (req, res, next)=> {
+  
+  const user = await User.findById(req.user.id);
+  if (!user) {
+    res.status(401);
+    throw new Error("Not authorized");
+  }
+  const {title} = req.body
```

Comment:
What happens if the req,body is empty? // Checking for empty input

Reply:
The front end will have validations. But I have added some error handling in the backend.